hotel-bookings-api/server.js

```diff
@@ -1,20 +1,63 @@
+process.env.PORT = process.env.PORT || 3001;
 const express = require("express");
 const cors = require("cors");
-
 const app = express();
-
 app.use(express.json());
 app.use(cors());
 
-//Use this array as your (in-memory) data store.
+//se this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
 app.get("/", function (request, response) {
   response.send("Hotel booking server.  Ask for /bookings, etc.");
 });
-
-// TODO add your routes and helper functions here
-
+app.get("/bookings", function (req, res) {
+  res.send({ bookings });
```

Comment:
You can change from `res.send({ bookings });` to` res.json({ bookings });`, it is not mandatory, but it's considered a good practice when you are specifically sending JSON data as a response.

Reply:
thanks for your review  Shadi,  I made changes :)